apps/crn-frontend/src/shared-research/export.ts

```diff
@@ -89,7 +89,9 @@ export const researchOutputToCSV = (
   created: output.created,
   lastModifiedDate: output.lastModifiedDate,
   published: output.published,
-  reviewRequestedBy: `${output.reviewRequestedBy?.firstName} ${output.reviewRequestedBy?.lastName}`,
+  statusChangedBy: `${output.statusChangedBy?.firstName} ${output.statusChangedBy?.lastName}`,
```

Comment:
This outputs `undefined undefined` for things which have not been in review (which will be 100% of research outputs initially).